air/src/trace/mod.rs

```diff
@@ -19,7 +19,7 @@ pub const MIN_TRACE_LEN: usize = 64;
 // ------------------------------------------------------------------------------------------------
 
 //      system          decoder           stack      range checks       chiplets
-//    (8 columns)     (24 columns)    (19 columns)    (3 columns)     (17 columns)
+//    (8 columns)     (24 columns)    (19 columns)    (3 columns)     (18 columns)
 // ├───────────────┴───────────────┴───────────────┴───────────────┴─────────────────┤
```

Comment:
I think this sums up to 72 columns. Is this also what gets printed out when we run benchmarks? (I think we may be off by 1 somewhere as before this change the number of columns that gets printed out was 70).

Reply:
71 is the true count, the culprit was the range checks column (is actually 2 columns, not 3).

Fixed the comment